src/objects.js

```diff
@@ -1,42 +1,53 @@
 const createPerson = (name, age) => {
-  // your code here
+  const person = {
+    name,
+    age
+  };
+return person;
```

Comment:
The return statement is missing indentation also you could just directly return the object, as you dont use it anywhere else you don't have to assign it to a variable:

`````
return {
    name,
    age
};
``````